README.md

```diff
@@ -8,6 +8,8 @@ TimeZones.jl
 [![Linux/OS X Build Status](https://travis-ci.org/JuliaTime/TimeZones.jl.svg?branch=master)](https://travis-ci.org/JuliaTime/TimeZones.jl)
 [![Windows Build Status](https://ci.appveyor.com/api/projects/status/ru96a9u8h83j9ixu/branch/master?svg=true)](https://ci.appveyor.com/project/omus/timezones-jl)
 [![codecov](https://codecov.io/gh/JuliaTime/TimeZones.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/JuliaTime/TimeZones.jl)
+[![](https://img.shields.io/badge/docs-stable-blue.svg)](http://timezonesjl.readthedocs.io/en/stable/)
+[![](https://img.shields.io/badge/docs-latest-blue.svg)](http://timezonesjl.readthedocs.io/en/latest/)
```

Comment:
Would be good to have some text in the `[]` for the seldom cases where the images don't load. Something like "Stable Documentation" and "Latest Documentation".

Reply:
I'll take care of this after.